fastlane/lib/fastlane/actions/sh.rb

```diff
@@ -1,3 +1,5 @@
+
+require_relative '../helper/sh_helper.rb'
```

Comment:
required, but not used?

Reply:
That might be some cruft left over from when I was working on special-casing the `sh` action. I'll take a look at that.